aws/data_source_aws_secretsmanager_secret.go

```diff
@@ -34,6 +35,11 @@ func dataSourceAwsSecretsManagerSecret() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"policy": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Since we are not providing `policy` as a configurable argument to lookup the secret, we should remove `Optional: true` here. 👍